livedata/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsControllerImpl.kt

```diff
@@ -89,20 +89,29 @@ internal class QueryChannelsControllerImpl(
 
     fun handleEvent(event: ChatEvent) {
         if (event is NotificationAddedToChannelEvent) {
+            // this is the only event that adds channels to the query
             addChannelIfFilterMatches(event.channel)
-        }
-        if (event is CidEvent) {
+        } else if (event is CidEvent) {
```

Comment:
`NotificationAddedToChannelEvent` is a `CidEvent` as well, why do we skip it?

Reply:
going to clarify this, it's confusing

Reply:
clarified which method calls update the results vs which ones update the channel state. that was confusing.